### DISCUSSION TITLE
How to Config the Image Drift?

### DISCUSSION THREAD

**Discussion by blue-j** [openseadragon/openseadragon#2024] [Q&A] [answered]
When I release the mouse, the image drifts for a moment more. Is there a config to turn that off or set the time it takes?  Thank you so much!

-J

**Comment by iangilman** [comment, marked as answer]
When dragging with the mouse OSD uses springs to create a smoother movement (though of course not everyone prefers this). When you let up on the mouse, the springs are still moving and take a moment to settle. One option you have is to modify or disable the springs using the `springStiffness` (must be greater than 0, bigger numbers make for a sharper end) and/or `animationTime` (in seconds, must be 0 or greater) options. This will affect all uses of springs, including zooming. 

Another option is to turn the "flick" feature on (which disables the springs for dragging) but then disable it. You would do that by passing this option when creating the viewer: 

```
            gestureSettingsMouse: { flickEnabled: true, flickMomentum: 0 },
```

Note that the behavior is different for touch (no springs during dragging, but flick momentum after dragging), since touch has different common expectations.
